wasi/wasi.go

```diff
@@ -82,9 +84,60 @@ type API interface {
 	// See https://en.wikipedia.org/wiki/Null-terminated_string
 	ArgsSizesGet(ctx wasm.HostFunctionCallContext, resultArgc, resultArgvBufSize uint32) Errno
 
-	// TODO: EnvironGet(ctx api.hostFunctionCallContext, environ, environBuf uint32) Errno
+	// EnvironGet is the WASI function that reads environment variables.
```

Comment:
link to the option that controls it, which should be named `Environ` for consistency both in wasi and also in go

```suggestion
	// EnvironGet is the WASI function that reads environment variables (Environ).
```